### ISSUE TITLE
Stork agent does not honor listen-only flags

### ISSUE THREAD

**Issue by ymartin-ovh** [isc-projects/stork#3]
Hello

It appears that despite --listen-only-* flags, Stork Agent server:
started serving Stork Agent                   address="0.0.0.0:8080"

Regards

**Comment by tomaszmrugalski** [comment]
Hi there!

This github repository is a clone of the main repo at https://gitlab.isc.org/isc-projects/stork. You'll get much faster responses, as the Stork developers use it daily. It's a pure coincidence that I've spotted it here.

Also, which stork version are you using? There was a related fix in this area (see [#504](https://gitlab.isc.org/isc-projects/stork/-/issues/504)) that was released in 0.16. It's not exactly fix for the problem you reported, but since there were changes in that area, it would be useful if we knew exactly the version. Also, there's slim, but non-zero chance that the problem was fixed in 0.16.

Can I ask you to go to the [main Stork repo](https://gitlab.isc.org/isc-projects/stork) and open the issue there?

I'm closing this issue, because we're shutting down the issue tracker on github.
